udata_front/theme/gouvfr/datagouv-components/src/components/SmallChart/SmallChart.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+    <canvas
+      ref="canvas"
+      width="120"
+      height="30"
+    ></canvas>
+    <div class="fr-grid-row justify-between">
+      <p class="text-xxs fr-m-0 text-mention-grey">{{ startDate }}</p>
+      <p class="text-xxs fr-m-0 text-mention-grey">{{ endDate }}</p>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Chart, type ChartOptions } from 'chart.js/auto';
+import { computed, onMounted, ref, useTemplateRef, watchEffect } from 'vue';
+import { formatDate } from '@datagouv/components/ts';
```

Comment:
The package can't import itself, the IDE is wrong with these autocompleted imports

Reply:
It's weird that it's working in local… But fixed in 8072adfa

Reply:
NPM has a weird fallback strategy, if it can't found a package in the current `node_modules`, it moves to the parent one.